configure.ac

```diff
@@ -655,6 +686,10 @@ expand_path_option mandir
 
 dnl Home-grown variables
 
+AS_IF([test x"${localedir}" = x""], [localedir="${datadir}/locale"])
```

Comment:
The above line is not necessary; expand_path_option will use its last argument as the default for the variable